### ISSUE TITLE
Report endpoint returns 500 error

### ISSUE THREAD

**Issue by yabirgb** [gitcoinco/web#9219]
**Describe the bug**
When querying information for our grant we get information if we don't use any extra argument but if we set the `from_timestamp` and the `to_timestamp` the server returns a 500 error 

```
{
    "active": "error",
    "code": 500,
    "nav": "home",
    "title": "Error 500"
}
```

**To Reproduce**
Call the url https://gitcoin.co/api/v0.1/grants/contributions_rec_report/?id=149&from_timestamp=2021-05-11&to_timestamp=2021-05-30&format=json

**Expected behavior**
Should return information from the 5th of May 2021 to the 30th of May 2021

**Desktop (please complete the following information):**

- OS: Solus OS

**Comment by yabirgb** [comment]
Followed the information in https://github.com/gitcoinco/web/pull/8883

**Comment by thelostone-mc** [comment]
seems to have missed defined the format var. throwing in fix at 4ceb23e3e73fa8c05b65bd6833dcc9274d4e54b2

**Comment by thelostone-mc** [comment]
This should be fixed now !
